templates/registry/markdown/entity_entry.md.j2

```diff
@@ -0,0 +1,16 @@
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/entity_entry.md.j2 -->
+{% import 'stability.j2' as stability %}
+{% import 'attribute_table.j2' as at %}
+
+# {{ ctx.name | title_case }}
+
+**Status:** {{ stability.badge(ctx.stability, ctx.deprecated) }}
+
+**type:** `{{ ctx.name }}`
+
+**Description:** {{ ctx.brief }}
+
+{# TODO: Add identifying vs. descriptive attributes #}
+{{ at.generate(ctx.attributes, "", "../attributes", ctx.lineage.attributes) }}
```

Comment:
wdyt about using absolute path from the repo root? I.e. `/docs/registry/attributes"

Reply:
Fixed, good suggestion!